docs/reference/discovery.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Discovery
+---
+
+## Discovery
+
+Knowing all potential wallets and wallet extensions available to user and that work on a blockchain can be challenging. Flow's Discovery is a way to take much of the burden of discovering and integrating with Flow compatible wallets off of the dapp developer so they can worry about building their app instead of wallet integrations and give as many options as possible to their users.
+
+There are two ways an app can use Discovery: our **UI version** which can be pointed to or the **API version** which allows you to access authentication services directly in your code.
+
+## UI Version
+
+If you've been to [Flow Port](https://port.onflow.org/), you've seen and/or interacted with the UI version of Discovery. It looks like this:
+
+Upon authenticating, a user is shown a list of services they can use to authenticate with Flow Port. Take a look below.
+
+![FCL Default Discovery UI](docs/images/discovery-ui.jpg)
+
+This method is simple to integrate into your app. FCL handles everything behind the scenes. All you have to do is point to Discovery in your configuration instead of a single wallet with configuration code like this:
+
+```javascript
+import { config } from "@onflow/fcl";
+
+config({
+  "accessNode.api": "https://access-testnet.onflow.org", // Mainnet: "https://access-mainnet-beta.onflow.org"
+  "discovery.wallet": "https://fcl-discovery.onflow.org/testnet/authn" // Mainnet: "https://fcl-discovery.onflow.org/authn"
+})
+```
+
+Once that configuration is set, any time you call `authenicate` using `fcl` the user will be presented with that screen.
+
+### Branding Discovery UI
+
+Starting in version 0.0.79-alpha.4, dapps now have the ability to display app a title and app icon in the Discovery UI by setting a few values in their FCL app config. Branding is important because it adds an element of trust to your site by showing an app icon and title that a user is before going on to take an action in a wallet.
+
+All you have to do is set `app.detail.icon` and `app.detail.title` like this:
+
+```javascript
+import { config } from "@onflow/fcl";
+
+config({
+  "app.detail.icon": "https://placekitten.com/g/200/200",
+  "app.detail.title": "Kitten Dapp"
+})
+```
+
+> ** Note: This screen will also appear if you call authorize on any transaction when a user is not logged in so it’s important to set this even if you are mainly using the following API version of Discovery as not setting these configurations may have your information show with a default icon and “Unknown App.” **
```

Comment:
** Note: If these configuration options aren't set, Dapps using the Discovery API will still display a default icon and “Unknown App” title when attempting to authorize a user who is not logged in.